django_netjsonconfig/base/template.py

```diff
@@ -100,5 +100,13 @@ def clean(self, *args, **kwargs):
         if self.type == 'vpn' and not self.config:
             self.config = self.vpn.auto_client(auto_cert=self.auto_cert)
 
+    def get_context(self):
+        c = {
+            'id': str(self.id),
+            'name': self.name,
+        }
+        c.update(super(AbstractTemplate, self).get_context())
```

Comment:
I added this line in end instead of beginning because if we change this dict to json we want `app_settings.CONTEXT` on end